Source/JavaScriptCore/wasm/WasmOMGIRGenerator32_64.cpp

```diff
@@ -6003,12 +5999,8 @@ auto OMGIRGenerator::addCall(unsigned callSlotIndex, FunctionSpaceIndex function
                 handle->generate(jit, params, this);
 
             ASSERT(!m_info.isImportedFunctionFromFunctionIndexSpace(functionIndexSpace));
-            RefPtr<IPIntCallee> callee;
-            {
-                Locker locker { m_calleeGroup.m_lock };
-                callee = m_calleeGroup.ipintCalleeFromFunctionIndexSpace(locker, functionIndexSpace);
-            }
-            jit.storeWasmCalleeToCalleeCallFrame(CCallHelpers::TrustedImmPtr(CalleeBits::boxNativeCallee(callee.get())), isTailCall ? sizeof(CallerFrameAndPC) - prologueStackPointerDelta() : 0);
+            Ref<IPIntCallee> callee = m_calleeGroup->ipintCalleeFromFunctionIndexSpace(functionIndexSpace);
```

Comment:
Should use `m_calleeGroup.` instead of `m_calleeGroup->`.

Reply:
Done.